app/helpers/admin_helper.rb

```diff
@@ -1,6 +1,6 @@
 module AdminHelper
-  def certificate_component_url(component, component_id)
-    component == 'SpComponent' ? sp_component_path(component_id) : msa_component_path(component_id)
+  def certificate_component_url(component_type, component_id)
+    component_type == 'SpComponent' ? sp_component_path(component_id) : msa_component_path(component_id)
```

Comment:
We have a constant for 'SpComponent' => COMPONENT_TYPE::SP

Reply:
Yes thank you, i thought we did but had trouble finding it